osf/models/preprint_provider.py

```diff
@@ -28,6 +28,7 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     social_twitter = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
```

Comment:
No. I think that, _technically_ the API won't be breaking, but if anyone is using the older version of the preprint_provider information, they'll quickly become sad.